dirty_cat/_minhash_encoder.py

```diff
@@ -155,6 +162,41 @@ def get_fast_hash(self, string: str) -> np.array:
                 ]
             )
 
+    def compute_hash(self, x) -> np.array:
+        """
+        Function called by joblib Parallel to compute the hash.
+
+        Check if the string is in the hash dictionary, if not, compute the hash using
+        the specified hashing function and add it to the dictionary.
+
+        Parameters
+        ----------
+        x : str
+            The string to encode.
+
+        Returns
+        -------
+        np.array of shape (n_components, )
+            The encoded string, using specified encoding scheme.
+        """
+        if x not in self.hash_dict_:
+            if x == "NAN":  # true if x is a missing value
+                self.hash_dict_[x] = np.zeros(self.n_components)
+            else:
+                if self.hashing == "fast":
+                    self.hash_dict_[x] = self.get_fast_hash(x)
+                elif self.hashing == "murmur":
+                    self.hash_dict_[x] = self.minhash(
+                        x,
+                        n_components=self.n_components,
+                        ngram_range=self.ngram_range
+                    )
+                else:
+                    raise ValueError("hashing function should be either 'fast' or"
+                                     "'murmur', got '{}'"
+                                     "".format(self.hashing))
```

Comment:
f-strings for the win :)
```suggestion
                    raise ValueError("hashing function should be either 'fast' or"
                                     f"'murmur', got '{self.hashing!r}'")
```